github/NamedUser.py

```diff
@@ -46,6 +46,14 @@ class NamedUser(github.GithubObject.CompletableGithubObject):
     def __repr__(self):
         return self.get__repr__({"login": self._login.value})
 
+    def __hash__(self):
+        return hash((self.id, self.login))
+
+    def __eq__(self, other):
+        return (isinstance(other, type(self))
```

Comment:
Thanks for the change. Even the most outer pair of brackets are not need.
```python
return isinstance(other, type(self)) and self.login == other.login and self.id == other.id)
```

Reply:
Ah! The linter was throwing an error, so I hesitated. Will make the change nevertheless

Reply:
What did the linter say?